chain/store/splitstore/snoop.go

```diff
@@ -0,0 +1,161 @@
+package splitstore
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/bmatsuo/lmdb-go/lmdb"
+
+	cid "github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/go-state-types/abi"
+)
+
+var TrackingStoreMapSize int64 = 1 << 34 // 16G
+
+type TrackingStore interface {
+	Put(cid.Cid, abi.ChainEpoch) error
+	PutBatch([]cid.Cid, abi.ChainEpoch) error
+	Get(cid.Cid) (abi.ChainEpoch, error)
+	Delete(cid.Cid) error
+	Keys(context.Context) (<-chan cid.Cid, error)
+	Close() error
+}
```

Comment:
Does this need to mimic the blockstore interface? I don't think so. Can simplify this to:

```go
type Tracker interface {
    // called on bs.Put, bs.PutMany, bs.Get
    Track(abi.ChainEpoch, cids... cid.Cid) error
    // called on delete (actually unused)
    Untrack(cids... cid.Cid) error
    // iterator
	GetTracked(context.Context) (<-chan cid.Cid, error)
    Close() error
}
```

?

Reply:
Not really, I defined the methods we need with their standard names.
I don't see what this new interface you propose adds.

Reply:
It adds simplicity and doesn't trick the reader into thinking that this is a blockstore.